dev-util/capstone/capstone-4.0.1.recipe

```diff
@@ -14,11 +14,11 @@ Solaris confirmed).
 * High performance & suitable for malware analysis (capable of handling \
 various X86 malware tricks)."
 HOMEPAGE="http://www.capstone-engine.org"
-COPYRIGHT="2013-2015, COSEINC"
+COPYRIGHT="2013-2018, COSEINC"
 LICENSE="BSD (3-clause)"
 REVISION="1"
 SOURCE_URI="https://github.com/aquynh/capstone/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="913dd695e7c5a2b972a6f427cb31f2e93677ec1c38f39dda37d18a91c70b6df1"
+CHECKSUM_SHA256="79bbea8dbe466bd7d051e037db5961fdb34f67c9fac5c3471dd105cfb1e05dc7"
```

Comment:
Some suggestions:
```diff
+srcGitRev="b01a30bf595c23081d0acfbcbe36fc8c6d829a71"
+SOURCE_URI_2="https://github.com/groundx/capstonefuzz/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256_2="837be0138cb44384bf1266e76d67947a5be30a1dbd8d8ea532491677c1609b78"
```

Reply:
Done.

Reply:
I don't know whether this is correct: it will be packaged in the _source package (possibly from another license). Test data shouldn't be needed to build this recipe.

Reply:
@korli, would a separate recipe like "capstone_test" do?